packages/hint/docs/user-guide/concepts/parsers.md

```diff
@@ -16,40 +16,35 @@ configuration file:
 
 ## List of official `parser`s
 
+* [`css`][@hint/parser-css]: A `CSS` parser built on top of
+  [PostCSS][postcss] so hints can analyze `CSS` files.
+
 * [`javascript`][@hint/parser-javascript]: A `JavaScript`
   parser built on top of `ESLint` so hints for analyzing `JavaScript`
   files can be built.
 
+* [`html`][@hint/parser-html]: An `HTML` parser built on top of `jsdom`.
```

Comment:
Did you intend to list the parsers alphabetically? If so, this should be before `javascript`.